taropsbt/address.go

```diff
@@ -0,0 +1,45 @@
+package taropsbt
+
+import (
+	"github.com/lightninglabs/taro/address"
+	"github.com/lightninglabs/taro/asset"
+)
+
+// FromAddress creates an empty virtual transaction packet from the given
+// address. Because sending to an address is always non-interactive, a change
+// output is also added to the packet.
+func FromAddress(receiverAddr *address.Taro) *VPacket {
+	pkt := &VPacket{
+		Input: &VInput{
+			PrevID: asset.PrevID{
+				ID: receiverAddr.ID(),
+			},
+		},
+		Outputs: make([]*VOutput, 2),
+	}
+
+	// If we are sending the full value of the input asset, or sending a
+	// collectible, we will need to create a split with un-spendable change.
+	// Since we don't have any inputs selected yet, we'll use the NUMS
+	// script key to avoid deriving a new key for each funding attempt.
+	pkt.Outputs[0] = &VOutput{
```

Comment:
Should actually be conditional? So then would only set the receiver output.

Reply:
No, not in this case. Since we're creating a PSBT for a send to an address, it will be a non-interactive send and therefore always have a split (but maybe a zero value with NUMS point).